### ISSUE TITLE
[BUG] autoexec regression

### ISSUE THREAD

**Issue by liPillON** [ZDoom/Raze#1085]
### Raze version

Raze 1.10.1

### Which game are you running with Raze?

Duke Nukem 3D

### What Operating System are you using?

Windows 10

### Please describe your specific OS version

Enterprise LTSC 2021 x64

### Relevant hardware info

intel i7 1260p, intel Iris XE 96EU

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

See the issue filed for GZDoom a while ago.
This regression propagated to vkdoom and raze with one of the subsequent backend updates.
It has been since fixed in GZ master/release and in VK master.

### Steps to reproduce the behaviour.

See the issue filed for GZDoom.

**Comment by madame-rachelle** [comment]
Is this bug still occurring?

**Comment by liPillON** [comment]
yup, just checked using the latest drd team build (1.11pre-36-g1f0f5a413)
